examples/c/dl/dl_example.rvm

```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
```

Comment:
Shall we replace this dummy example with the waterTank example?

Reply:
Yes, that would be good.